### ISSUE TITLE
el-breadcrumb-item to not work

### ISSUE THREAD

**Issue by npm-ued** [element-plus/element-plus#923]
code like this 
``` 
<el-breadcrumb-item :to="{ path: '/' }">首页</el-breadcrumb-item>
<el-breadcrumb-item :to="{ path: '/test' }">活动管理</el-breadcrumb-item>
```
vue 3.0.3
vue-router 4.0.0-rc.6
element-plus 1.0.1-beta.0
not call vue-router

**Comment by zazzaz** [comment]
it is better to try the latest version of element-plus to see if it has been fixed

**Comment by JeremyWuuuuu** [comment]
> code like this
> 
> ```
> <el-breadcrumb-item :to="{ path: '/' }">首页</el-breadcrumb-item>
> <el-breadcrumb-item :to="{ path: '/test' }">活动管理</el-breadcrumb-item>
> ```
> 
> vue 3.0.3
> vue-router 4.0.0-rc.6
> element-plus 1.0.1-beta.0
> not call vue-router

Try the latest version, it should be fixed in #745

**Comment by JeremyWuuuuu** [comment]
Closing since this was fixed in the latest version
